### ISSUE TITLE
[8.0.0-beta.2/8.0.0] Error in console when removing row or column when columnSummary is set

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#7174]
### Description
Removing row or column when column summary is set with `destinationRow` and `destinationColumn` causes an error in the browser console

 _"Error: Assertion failed: Expecting an unsigned number."_ on 

<img width="422" alt="Screenshot 2020-08-06 at 14 12 22" src="https://user-images.githubusercontent.com/30186818/89530506-e86f0680-d7ee-11ea-8079-50e3ad026760.png">

```
function assert(condition, errorMessage) {
  if (!condition()) {
    throw new Error("Assertion failed: ".concat(errorMessage));
```

### Steps to reproduce

1. Remove any row or column.

### Demo

https://jsfiddle.net/aninde/cs0e6g9d/ - v8.0.0-beta.2 (**broken**)
https://jsfiddle.net/aninde/w174k0su/ - v8.0.0 (**broken**)
https://jsfiddle.net/aninde/7qr4gtjo/ - v8.0.0-beta.1 (**fixed**)
https://jsfiddle.net/aninde/k3d2L6j9/ - v7.4.2 (**fixed**)

### Your environment
* Handsontable version: 8.0.0-beta.2  &  8.0.0
* Browser Name and version: Firefox 79, Chrome 84
* Operating System: macOS Catalina 10.15.6

**Comment by AMBudnik** [comment]
v8.0.0 example https://jsfiddle.net/zj6cuwm2/ works the same (ps. I've added numeric data to avoid `NaN`)

**Comment by aninde** [comment]
This issue will be fixed by the upcoming `v8.1.0` https://jsfiddle.net/aninde/cpgn51o9/.

**Comment by AMBudnik** [comment]
Version 8.1.0 has just been released. We can close the issue, the regression is no longer replicable.
